services/terrain/src/terrain/services/fileio/controllers.clj

```diff
@@ -48,10 +44,8 @@
      req  - the ring request map"
   [{:keys [user dest]} ^IPersistentMap req]
   (let [store                   (partial store-from-form user dest)
-        {{file "file"} :params} (multipart/multipart-params-request req {:store store})]
-    (when-not file
-      (throw+ {:error_code ERR_MISSING_FORM_FIELD :field "file"}))
-    (success-response {:file (data/path-stat user file)})))
+        {{file-info "file"} :params} (multipart/multipart-params-request req {:store store})]
```

Comment:
The value that the middleware's store function returns ends up in the key relating to the part name, i.e. 'file'; so, file-info here is the data-info response, which matches the former return value.